ci/jenkins/test.sh

```diff
@@ -163,11 +163,18 @@ function export_govc_env_var {
 
 function clean_antrea {
     echo "====== Cleanup Antrea Installation ======"
-    clean_up_one_ns "monitoring"
-    clean_up_one_ns "antrea-ipam-test-11"
-    clean_up_one_ns "antrea-ipam-test-12"
-    clean_up_one_ns "antrea-ipam-test"
-    clean_up_one_ns "antrea-test"
+    clean_ns "monitoring"
+    clean_ns "antrea-ipam-test-11"
+    clean_ns "antrea-ipam-test-12"
+    clean_ns "antrea-ipam-test"
+    clean_ns "antrea-test"
+    echo "====== Cleanup Conformance Namespaces ======"
+    clean_ns "net"
```

Comment:
How did these namespaces create? By running e2e test or conformance test? If it is from e2e, shall we add cleanup in the test code?

Reply:
The goal is to delete conformance namespaces starting with "net", like netpol-xx-xx.

Reply:
It seems the question is unanswered properly, did we observe the namespace were left over after tests finished? and do we know why?

Reply:
Got it.